.eslintrc.json

```diff
@@ -16,15 +15,9 @@
         "parser": "@typescript-eslint/parser",
         "project": "./tsconfig.json"
       },
-      "overrides": [
-        {
-          "files": ["vite.config.ts"],
-          "rules": {
-            "import/no-default-export": "off"
-          }
-        }
-      ],
+
       "rules": {
+        "@typescript-eslint/no-explicit-any": "warn",
```

Comment:
Why did you change this rule?

Reply:
Because it throws a lot of errors. I don't think it worked previously. We can fix this, but maybe in another PR?

Reply:
Ah, I think NX used to set this rule before. Fine to add it imo!